webviz_config/testing/_plugin.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from ._composite import WebvizComposite
+
+# pylint: disable=too-few-public-methods
+class MissingWebvizTesting:
+    def __init__(self, **kwargs):
+        raise Exception(
```

Comment:
Should we instead raise a `RuntimeError`? 

https://docs.python.org/3/library/exceptions.html#RuntimeError